full-service/src/service/models/tx_proposal.rs

```diff
@@ -547,6 +547,7 @@ mod tests {
                     subaddress_index: Some(alice_address_from_bob.subaddress_index as u64),
                 },
                 None,
+                None,
```

Comment:
Not testing spend_only_from_subaddress at the tx_proposal level, as this is simply testing the construction of the tx_proposal is sound, not whether it selected the correct TXOs. That logic is tested in txo.rs